src/api.raml

```diff
@@ -1842,6 +1901,20 @@ traits:
                                 description: |
                                     Demonstration of the list being empty (the first page will always return a valid result).
                                 value: !include samples/search/v1/empty.json
+                    application/vnd.elife.search+json;version=2:
+                        (experimental):
+                        schema: !include ../dist/model/search.v2.json
+                        examples:
```

Comment:
`paginated` and `empty` examples are pretty useless.

What would be *much* more helpful is a `description` of how `v2` differs from `v1`, especially if clients consuming search results need to migrate.